### PR TITLE
Fix: Farmer Form Image Inputs

### PR DESCRIPTION
**Pull request by AlexEntrepreneur**
# Description

Fix errors/bugs with form image inputs

## Checklist

Remove any items which are not applicable.

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works AND the tests pass

### CODE REVIEW

File: src/components/pages/UpdateFarmer/UpdateFarmer.js [hunk 1]
```diff
@@ -97,17 +94,40 @@ const UpdateFarmer = ({ location, history, appStateShouldUpdate, user }) => {
...
+        const imageFile = new FormData();
+        imageFile.append('file', files[0]);
+        imageFile.append(
+          'upload_preset',
+          process.env.REACT_APP_CLOUDINARY_PRESET
+        );
+
+        try {
+          if (process.env.REACT_APP_CLOUDINARY_URL) {
+            const uploadResponseData = await axios.post(
```
Comment (daquinons) [inline src/components/pages/UpdateFarmer/UpdateFarmer.js]: Same than in previous requests, an `imageUploadHandler` could be good.

File: src/components/pages/UpdateFarmer/UpdateFarmer.spec.js [hunk 2]
```diff
@@ -108,3 +109,17 @@ describe('Update Farmer component', () => {
...
+
+  const FarmerForm = render(
+    <Router>
+      <UpdateFarmer location={{ state: { farmer: farmerMock } }} />
+    </Router>
+  );
+
+  const farmerFormImage = FarmerForm.getByAltText('Farmer Form Image');
+  
+  expect(farmerFormImage.src).toEqual(farmerMock.personalInfo.image_url);
```
Comment (daquinons) [inline src/components/pages/UpdateFarmer/UpdateFarmer.spec.js]: Tests 👍

File: src/components/pages/AddFarmer/AddFarmer.js [hunk 3]
```diff
@@ -72,17 +72,43 @@ const AddFarmer = () => {
...
+        
+        const imageFile = new FormData();
+        imageFile.append('file', files[0]);
+        imageFile.append(
+          'upload_preset',
+          process.env.REACT_APP_CLOUDINARY_PRESET
+          );
+          try {
+            if (process.env.REACT_APP_CLOUDINARY_URL) {
+              const uploadResponseData = await axios.post(
```
[the remark this answers is not in the document]
Comment (AlexEntrepreneur) [reply src/components/pages/AddFarmer/AddFarmer.js]: Yes, this has already been addressed in another card and is not the purpose of the PR. 👍

File: src/components/common/Input/Input.js [hunk 4]
```diff
@@ -5,6 +5,7 @@ import {
   Checkbox,
   Form
 } from 'semantic-ui-react';
+import { ImageInputContainer } from './ImageInputContainer';
```
[the remark this answers is not in the document]
Comment (AlexEntrepreneur) [reply src/components/common/Input/Input.js]: Thanks!

File: src/components/pages/UpdateFarmer/UpdateFarmer.js (hunk 1 above)
Comment (AlexEntrepreneur) [reply src/components/pages/UpdateFarmer/UpdateFarmer.js]: Will be addressed in another PR. 👍

**Pull request merged**